modules/mobile/app/controllers/mobile/v1/messages_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V1
+    class MessagesController < MessagingController
+      include Filterable
```

Comment:
It looks like you copied this over from V0, but in V0 it's being used to support this `resource.find_by(filter_params)` and there doesn't seem to be any equivalent going on here. Are we using Filterable anywhere here?

Reply:
Good catch, no we're not